### DISCUSSION TITLE
How can I try Loop 2.0 Alpha?

### DISCUSSION THREAD

**Discussion by lukaszlew** [iSoron/uhabits#870] [General Discussion]
I can't install f-droid because corp policy prevents it.
I wonder whether I could create a new app on Play store (just for myself), and try to upload it. 

Is there a better way? 
And if so, will it work?

(My central motivation is that if I fail a work habit during the week, the habit keeps bugging me during the weekend, and I'm hoping that 2.0 will fix that. )

**Comment by iSoron** [comment]
Hi @lukaszlew, the alpha version is not available on F-Droid either. The only way to install it currently is to donwnload the APK [from our releases page](https://github.com/iSoron/uhabits/releases/tag/v2.0.0-alpha).

> the habit keeps bugging me during the weekend

When you create a reminder, you can set it to only appear during week days. This has been available since the early versions of the app. Does that fix the problem?

**Reply by lukaszlew** [reply]
Hey Álinson, Thanks for your answer. 
I did my due dilligence and I have read the FAQ and searched a lot of issues in the past and have not found support for my workflow. Unfortunately reminder configuration does not fix my problem. The root cause seems to me that while Loop supports reminders very well, my workflow does not use reminders at all.

My meta-habit is to check the main screen of Loop and see what's left to do for me today.
I must highlight that it is extremely motivating to check off things and make the list go smaller and go to sleep with empty screen. (similar to Inbox Zero routine and GTD)

My current workaround is to actually mark 'work habits' (which are configured as 5x a week) as done first thing on Saturday morning if I failed them once during the week.

**Comment by drawtheliner** [comment]
I have two questions regarding Loop 2.0.

1. If I decide to try out the alpha apk, does it replace the current app, or would it run as a completely separate app (without shared data)?
2. Is there an estimate for release date? I know there's no way to know for sure. But maybe a min/max range? :)

**Comment by ke352802081770314** [comment]
I was under the impression that 2.0 is the current version and already in general release. Personally, I use LineageOS without Google services, so F-Droid or apk download sites are my source of software, and in this case I downloaded from F-Droid that gave no indication of beta or showed a recommended lower version. 
So I am really surprised that https://play.google.com/store/apps/details?id=org.isoron.uhabits&hl=de&gl=US still lists 1.8.,12 as the current version, and that in #964 2.0 was recently stated to be beta. But you may find a Google Play method for trying 2.0 explained in the comment there.

**Reply by iSoron** [reply]
@ke352802081770314 This question was originally about Loop 2.0 Alpha. That version was only available on GitHub Releases. You can get the beta version by [joining our open beta on Google Play](https://play.google.com/apps/testing/org.isoron.uhabits) or downloading it from [F-Droid](https://f-droid.org/).

We don't tag our releases as beta/stable because **all** of our releases (including patch releases) go through beta testing first, and are only released to all users if no serious bugs are found. Loop 1.8.12, for example, was a beta version for a while some time ago, but was deemed stable enough to be released to all users.
